algorithm/WebRelated/preventPaste/prevent.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Fungsi untuk melarang user mem-paste di inputan form website kita.
+ * Fungsi ini akan mengembalikan alert javascript "Dilarang mem-paste!" jika user melakukan paste pada sebuah inputan form.
+ */
+
+// Mengambil elemen input yang akan dilarang mem-paste.
+const input = document.querySelector("input");
+// Membuat event listener jika user melakukan paste pada inputan form.
```

Comment:
Dikasih jarak buat penulisannya

```suggestion
const input = document.querySelector("input");

// Membuat event listener jika user melakukan paste pada inputan form.
```

Reply:
Oke, sudah saya perbaiki